### DISCUSSION TITLE
안녕하세요 닷트윈 질문입니다

### DISCUSSION THREAD

**Discussion by hjs6082** [naverz/zepeto-studio-kor#1079] [5. Q&A]
제페토 스크립트에서 닷트윈 사용 가능한가요?
아니면 C# 스크립트와 연동하는 방법이 혹시 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요 현재는 닷트윈을 지원하지 않습니다.
추후에 지원할 수 있도록 내부에서 검토해 보도록 하겠습니다~
감사합니다!
